coverage-targets.js

```diff
@@ -6,10 +6,10 @@
 // subset of files to check against these targets.
 module.exports = {
   global: {
-    lines: 62.25,
-    branches: 50.5,
-    statements: 61.5,
-    functions: 55,
+    lines: 68.73,
+    branches: 47.82,
+    statements: 66.99,
+    functions: 62.45,
   },
```

Comment:
Was getting this error

```
Error: Coverage thresholds for global NOT met
Lines: 4364/6349 (68.73%). Target: 62.25%
Branches: 1737/3632 (47.82%). Target: 50.5%
Statements: 4452/6645 (66.99%). Target: 61.5%
Functions: 800/1281 (62.45%). Target: 55%
```

And then I was getting this error 

```
Error: Coverage EXCEEDS threshold for global and must be bumped
Lines: 4364/6349 (68.73%). Target: 62.25%
Branches: 1737/3632 (47.82%). Target: 47.82%
Statements: 4452/6645 (66.99%). Target: 61.5%
Functions: 800/1281 (62.45%). Target: 55%
```

cc @tmashuang could you please advise if this is right?

Reply:
Could you do me a favor and remove your changes to this file? I'm going to pull your branch locally and figure out what the problem is. Nothing you did should have resulted in this drastic of a change in coverage

Reply:
Rebased and removed this update. Seems like it's passing 

<img width="909" alt="Screenshot 2023-02-09 at 12 35 05 PM" src="https://user-images.githubusercontent.com/8112138/217930954-7938bd25-91d3-4a38-b17c-3cd0af40faf4.png">